network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -30,6 +35,16 @@
     image_field = ImageChooserPanel('image')
     MEDIA_URL = settings.MEDIA_URL
 
+vote_throttle_class = UserVoteRateThrottle if not settings.TESTING else TestUserVoteRateThrottle
+
+
+class ProductPageVoteBin(models.Model):
+    bin_1 = models.IntegerField(default=0, help_text='Total votes for 0%-20%')
+    bin_2 = models.IntegerField(default=0, help_text='Total votes for 21%-40%')
+    bin_3 = models.IntegerField(default=0, help_text='Total votes for 41%-60%')
```

Comment:
Simplified voting "bins". See help text for details.

Reply:
This feels like a job for int_list_validator, https://stackoverflow.com/questions/1429293/storing-an-integer-array-in-a-django-database/52497141